docs/cases/cases-overview.asciidoc

```diff
@@ -74,6 +75,24 @@ To view a case, click on its name. You can then:
 
 NOTE: Comments can also contain Markdown syntax and Timeline links.
 
+[float]
+[[cases-summary]]
+== Review the case summary
+
+Click on an existing case to open it and to access the case summary. The case summary, located under the case title, consists of metrics that summarize alert information and response times. The metrics update when you attach additional unique alerts to the case, add more connectors, or modify the case's status:
+
+* **Total alerts**: Shows the total number of unique alerts attached to the case.
+* **Associated users**: Shows the total number of unique users that are represented in the attached alerts.
+* **Associated hosts**: Shows the total number of unique hosts that are represented in the attached alerts.
+* **Total connectors**: Shows the total number of connectors that have been added to the case.
+* **Case created**: Shows the date and time that the case was created.
+* **Open duration**: Shows how much time has passed since the case was created.
+* **In progress duration**: Shows how long the case has been in the `In progress` state.
+* **Duration from creation to close**: Shows how much time has passed from when the case was created to when it was closed.
```

Comment:
Possible revisions to remove some of the repetition, which can get a little hard to quickly skim through.
```suggestion
* **Total alerts**: Total number of unique alerts attached to the case.
* **Associated users**: Total number of unique users represented in the attached alerts.
* **Associated hosts**: Total number of unique hosts represented in the attached alerts.
* **Total connectors**: Total number of connectors added to the case.
* **Case created**: Date and time that the case was created.
* **Open duration**: Time passed since the case was created.
* **In progress duration**: How long the case has been in the `In progress` state.
* **Duration from creation to close**: Time passed from when the case was created to when it was closed.
```